lib/Yam/ControlCenterPage.pm

```diff
@@ -0,0 +1,428 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Control Center
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::ControlCenterPage;
+use parent 'Yam::PageBase';
+use strict;
+use warnings;
+use testapi;    # needed because of bsc1206966
+
+sub init {
+    my $self = shift;
+    $self->{lbl_yast_cc} = $self->{app}->label({label => 'YaST Control Center'});
+    $self->{slb_groups} = $self->{app}->selectionbox({id => 'groups'});
+    $self->{slb_progs} = $self->{app}->selectionbox({id => 'progs'});
+    $self->{btn_help} = $self->{app}->button({id => 'help'});
+    $self->{btn_run} = $self->{app}->button({id => 'run'});
+    $self->{btn_quit} = $self->{app}->button({id => 'quit'});
+    return $self;
+}
+
+sub get_control_center_page {
+    my ($self) = @_;
+    die 'YaST Control Center is not displayed' unless $self->{btn_run}->exist();
+    return $self;
+}
+
+# workarounds
+
+sub apply_workaround_bsc1206966 {
+    my ($self) = @_;
+    record_soft_failure('bsc#1206966 - Item selection with REST API does not update contents');
+    my $currentgroup = $self->{slb_groups}->selected_items();
+    for (1 .. 4) { send_key 'tab'; }
+    if ($currentgroup eq 'Software') {
+        send_key 'down';
+        send_key 'up';
+    }
+    else {
+        send_key 'up';
+        send_key 'down';
+    }
+}
+
+# private methods collection
+
+sub _select_software {
+    my ($self) = @_;
+    $self->{slb_groups}->select('Software');
+    apply_workaround_bsc1206966();
+    return $self;
+}
+
+sub _select_online_update {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Online Update');
+    return $self;
+}
+
+sub _select_software_management {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Software Management');
+    return $self;
+}
+
+sub _select_add_on_products {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Add-On Products');
+    return $self;
+}
+
+sub _select_media_check {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Media Check');
+    return $self;
+}
+
+sub _select_software_repositories {
+    my ($self) = @_;
+    $self->{slb_progs}->select('Software Repositories');
+    return $self;
```

Comment:
you only need `$self` after methods that click on menus on the left, "software", "system", "hardware", etc, not the submenus, once you click in submenu you are in a different screen, so  you cannot chain any command.